hive-plugin/src/main/java/com/wipro/ats/bdre/hiveplugin/hook/LineageHook.java

```diff
@@ -29,56 +29,54 @@
 	private static final Logger LOGGER = Logger.getLogger(LineageHook.class);
 	private boolean successFlag;
 	public boolean isSuccess() {
-	return successFlag;
+		return successFlag;
 	}
 
 	@Override
 	public void run(HookContext hookContext) {
-	LOGGER.debug("Running LineageHook...");
-	String wholeQuery = null;
-	String processId = null;
-	String instanceExecId = null;
 
-	try {
-	if (hookContext == null) {
-		LOGGER.warn("HookContext is null. Assuming default values");
-		wholeQuery = LineageConstants.query;
-		processId = "" + LineageConstants.processId;
-		instanceExecId = "" + LineageConstants.instanceId;
+		LOGGER.info("Running LineageHook...");
+		String wholeQuery = null;
+		String processId = null;
+		String instanceExecId = null;
 
-	}
-	else {
-		wholeQuery = hookContext.getQueryPlan().getQueryString();
+	    try {
+		    if (hookContext == null) {
+			    LOGGER.warn("HookContext is null. Assuming default values");
+			    wholeQuery = LineageConstants.query;
+			    processId = "" + LineageConstants.processId;
+			    instanceExecId = "" + LineageConstants.instanceId;
 
-		Properties properties = hookContext.getConf().getAllProperties();
-		for (Map.Entry entry : properties.entrySet()) {
-			String key = (String)(entry.getKey());
-			if (LineageConstants.processIdString.equalsIgnoreCase(key)) {
-				processId = (String) (entry.getValue());
-				LOGGER.info("ProcessId found in Hive = " + processId);
-			}
-			else if (LineageConstants.instanceExecIdString.equalsIgnoreCase(key)) {
-				instanceExecId = (String) (entry.getValue());
-				LOGGER.info("InstanceExecId found in Hive = " + instanceExecId);
-			}
-		}
+		    } else {
+			    wholeQuery = hookContext.getQueryPlan().getQueryString();
 
-		if (processId == null || instanceExecId == null) {
-			LOGGER.warn("Warning: ProcessId or InstanceExecId is null. Lineage not done. End of Lineage Hook.");
-			return;
-		}
+			    Properties properties = hookContext.getConf().getAllProperties();
+			    for (Map.Entry entry : properties.entrySet()) {
+				    String key = (String)(entry.getKey());
+				    if (LineageConstants.processIdString.equalsIgnoreCase(key)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)